org.simondalvai.ball2box.yml

```
@@ -0,0 +1,35 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: '3.5'
+command: godot-runner
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+modules:
+  - name: Ball2Box
+    buildsystem: simple
+    build-commands:
+      # The game itself
+      - install Ball2Box.pck /app/bin/godot-runner.pck
+      # Metadata and a .desktop file
+      - install -D org.simondalvai.ball2box.desktop /app/share/applications/org.simondalvai.ball2box.desktop
+      - install -D org.simondalvai.ball2box.appdata.xml /app/share/metainfo/org.simondalvai.ball2box.metainfo.xml
+      # Icons
+      - install -D icon-256.png /app/share/icons/hicolor/256x256/apps/org.simondalvai.ball2box.png
+    sources:
+      # The game is pulled from GitHub
+      - type: archive
+        url: https://github.com/dulvui/ball2box/releases/download/v4.1.0/ball2box.tar.xz
+        sha256: 971b4a47bda0aa3f0bc6e1766b61eb848d0e10156505fef1f66531ac375bd5d4
```

```suggestion
      - type: archive
        strip-components: 0
        url: https://github.com/dulvui/ball2box/releases/download/v4.1.0/ball2box.tar.xz
        sha256: 971b4a47bda0aa3f0bc6e1766b61eb848d0e10156505fef1f66531ac375bd5d4
        
```

Should fix the archive content not being found